app/javascript/app/pages/login/forgot-password/forgot-password.js

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux';
+import { requestNewPassword } from '../../../services/login.service';
+import Component from './forgot-password-component';
+import { navigateToLogin } from './forgot-password-actions';
+
+const handleForgotPasswordClick = email => dispatch => {
+  requestNewPassword(email).then(ok => {
+    if (ok) dispatch(navigateToLogin());
+  });
+};
+
+const mapStateToProps = () => ({
```

Comment:
isn't this `mapDispatchToProps`? also you don't need also the dispatch boilerplate using `redux-tools` here https://github.com/Vizzuality/cw-ndc-tracking/pull/43/files#diff-dae511451e2f4c65d25f408ac803cfeaR4